elementary/utils/log.py

```diff
@@ -35,8 +37,21 @@ def get_console_handler():
     return console_handler
 
 
+def get_log_path():
```

Comment:
See my note above regarding getting the target_path as a parameter, I think it can simplify this function